src/EloquentDataTable.php

```diff
@@ -169,19 +169,23 @@ protected function joinEagerLoadedColumn($relation, $relationColumn)
 
                 case $model instanceof BelongsTo:
                     $table     = $model->getRelated()->getTable();
+                    $alias     = "alias_{$index}_{$table}";
+                    $tableAs   = "{$table} AS $alias";
```

Comment:
This will break some installations with Oracle database. Please use `"{$table} $alias"` removing the `AS` from query.

Reply:
But if i remove the `AS` then MySQL won't work...

Reply:
Can you run your tests again now?

Reply:
Sorry but you previous PR version is much better. AFAIK, it should work on MySQL since that is one of the reasons why I created this package version to work on Oracle.

On the other hand, I think I already created a patch that properly translates Oracle `AS` query. Will review with my other package https://github.com/yajra/laravel-oci8 too.

Reply:
So, what now? You want the first one again?